### PR TITLE
Test helper

### PR DESCRIPTION
**Pull request by lofi-nowhale**
## Description
This PR sets up our starter data for our test_helper file as well as our seeds.rb file so that we can start with data to test with and to seed our database. 

## Type of Change

- [ ] **fix**
- [ ] **feat**
- [x] **test**
- [ ] **refactor**
- [ ] **docs**

## Checklist

- [x] **Code has been self reviewed**
- [x] **Code runs without any errors**
- [x] **Thorough testing has been implemented if adding feature**
- [x] **All tests pass**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,31 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+Pet.destroy_all
+Shelter.destroy_all
+
```
Comment (ttakahashi1591) [inline db/seeds.rb:15]: Should we add here on line 11? 
 
- `ApplicationPet.destroy_all`
- `Application.destroy_all`
- `Review.destroy_all`

File: spec/test_helper.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  
+  @yuka = @shelter_4.pets.create!(name: 'Yuka', breed: "great dane", age: 4, adoptable: true)
+  @gonzo = @shelter_4.pets.create!(name: 'Gonzo', breed: "lab", age: 1, adoptable: false)
+  
+  @cherry = @shelter_5.pets.create!(name: 'Cherry', breed: "german shepard", age: 4, adoptable: true)
+  @portia = @shelter_5.pets.create!(name: 'Portia', breed: "chow chow", age: 1, adoptable: false)
+  
+  @stinky = @shelter_6.pets.create!(name: 'Stinky', breed: "australian shepard", age: 1, adoptable: false)
+  @bob = @shelter_6.pets.create!(name: 'Bob', breed: "blue heeler", age: 1, adoptable: false)
+end
```
Comment (ttakahashi1591) [inline spec/test_helper.rb]: Thanks for getting this added!!

**Review by ttakahashi1591 — CHANGES_REQUESTED** [review]

File: db/seeds.rb (hunk 1 above)
Comment (lofi-nowhale) [reply db/seeds.rb:15]: Ooh i think we probably should! Want to add these when we connect via zoom after office hours?

Comment (lofi-nowhale) [reply db/seeds.rb:15]: @ttakahashi1591  updated!

Comment (ttakahashi1591) [reply db/seeds.rb:15]: Perfect thanks!

**Review by ttakahashi1591 — APPROVED** [review]
This all looks great and I am glad we have our `test_helper` built out!

**Pull request merged**